dataland-frontend/tests/sharedUtils/components/UploadDocuments.ts

```diff
@@ -16,7 +16,6 @@ export const uploadDocuments = {
   },
   numberOfReportsToUploadShouldBe(expectedNumberOfReportsToUpload: number): void {
     cy.get('[data-test="report-to-upload-form"]').should("have.length", expectedNumberOfReportsToUpload);
-    cy.get('[data-test="report-to-upload-form"]').should("have.length", expectedNumberOfReportsToUpload);
   },
```

Comment:
Why have you committed fake fixtures for all frameworks?
This increases the probability of creating new test-fails.
You should rollback all fixture files, which are not related to this story (or are not affected by it). That's probably everything but sfdr fixtures, right?

Reply:
But why? Check this run:
https://github.com/d-fine/Dataland/actions/runs/5799141259/job/15718652833
The only test failing is the new created one. So i don't a reason to rollback fake fixtures.

Reply:
https://github.com/d-fine/Dataland/actions/runs/5811091520/job/15754995192
CI is green with the exception of sonar

Reply:
Ok if the CI is currently green, then let's leave it like that.  
It was more of a general suggestion not to commit unrelated fixtures, because it has no real advantage, but yields the risk of random things happening. But since it has already been done and everything is ok, I will close this.

Reply:
Agree with that.